packages/types-dev/objects.d.ts

```diff
@@ -1020,10 +1020,18 @@ declare global {
             common?: Partial<InstanceCommon>;
         }
 
-        /** TODO: To be defined */
-        type NotificationCategory = any;
+        type NotificationCategory = {
+            category: string;
+            name: Translated;
+            description: Translated;
+            /** `info` will only be shown by admin, while `notify` might also be used by messaging adapters, `alert` ensures both */
```

Comment:
This does not match how it is lived with our approach that notificaiton manager is used to send notifications all could be send in general. It is the importance of a notification and how adapters handle this is up to them. E.g. in notification manager some may decide to just get alerts via Mail which would represent real issues with the installation, while notify and info are kind of informative and info being the most optional.

Reply:
The description was copied from somewhere.

Reply:
What should we write there now?

Reply:
In another PR we had `/** Allows to define the severity of the notification with `info` being the lowest `notify` representing middle priority, `alert` representing high priority and often containing critical information */`